tensorflow/java/src/gen/perl/tftypes.pl

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/perl
```

Comment:
Python would probably look a bit cleaner. I'm not Pythonic enough to do the conversion easily, though.